gui/WorldView.cxx

```diff
@@ -1013,10 +1017,18 @@ void WorldView::loadMesh(QString const& path)
   vtkNew<vtkPLYReader> reader;
   reader->SetFileName(qPrintable(path));
   reader->Update();
-
+  
```

Comment:
Oops.  It looks like I introduced whitespace in the merge conflict resolution.  If someone wants to fix this please do.  Otherwise, I will get to this some time later.